Paper.tex

```diff
@@ -705,9 +776,27 @@ \subsection{Execution}
 $\hyperlink{G__accesslistaddress}{G_{\mathrm{accesslistaddress}}}$ and $\hyperlink{G__accessliststorage}{G_{\mathrm{accessliststorage}}}$ are the costs of warming up account and storage access, respectively.
 $G$ is fully defined in Appendix \ref{app:fees}.
 
+\hypertarget{effective_gas_price_p}{}We define the \textit{effective gas price}, formally $p$, as the amount of wei the transaction signer will pay per unit of gas consumed during the transaction's execution. It is calculated as follows:
+
+\begin{equation}
+  p \equiv \begin{cases}
+  T_{\mathrm{p}} & \text{if} \; T_{\mathrm{x}} = 0 \lor T_{\mathrm{x}} = 1 \\
+  f + H_{\mathrm{f}} & \text{if} \; T_{\mathrm{x}} = 2 \\
+  \end{cases}
+\end{equation}
+
+\hypertarget{priority_fee_f}{}where $f$ is the \textit{priority fee}---the amount of wei the block's beneficiary address will receive per unit of gas consumed during the transaction's execution. It is calculated as:
+
+\begin{equation}
+  f \equiv \begin{cases}
+  T_{\mathrm{p}} - H_{\mathrm{f}} & \text{if} \; T_{\mathrm{x}} = 0 \lor T_{\mathrm{x}} = 1 \\
+  min(T_{\mathrm{f}}, T_{\mathrm{m}} - H_{\mathrm{f}}) & \text{if} \; T_{\mathrm{x}} = 2 \\
+  \end{cases}
+\end{equation}
+
 The up-front cost $v_0$ is calculated as:
 \begin{equation}
-v_0 \equiv \hyperlink{tx_gas_limit_T__g}{T_{\mathrm{g}}} \hyperlink{tx_gas_price_T__p}{T_{\mathrm{p}}} + \hyperlink{tx_value_T__v}{T_{\mathrm{v}}}
+v_0 \equiv \hyperlink{tx_gas_limit_T__g}{T_{\mathrm{g}}} p + \hyperlink{tx_value_T__v}{T_{\mathrm{v}}}
```

Comment:
It should be `Tm` instead of `p` here: see https://github.com/ethereum/EIPs/pull/3594

Reply:
updated!